### DISCUSSION TITLE
Ability to add a list of proper nouns

### DISCUSSION THREAD

**Discussion by Vilhelm-Ian** [mortii/anki-morphs#18] [General]
I am currently learning german so I can't filter out cards by proper nouns like it's possible with the japanese addon. A possible solution is for me to be able to add a list of proper nouns and Anki-Morphs to automatically skip them.

If spacy is implemented there might be not a need for this

**Comment by aleksejrs** [comment]
MorphMan supports a database of "known" morphs, which is filled from the cards with a special tag, so you can put proper nouns in a card outside of your deck.

**Comment by Vilhelm-Ian** [comment]
thanks. That's a workaround I can do. But still I think it should atleast be considered as a feature.

**Comment by Vilhelm-Ian** [comment]
If i added a card with 10_000 names . My Known Morphs will jump by 10_000

**Comment by mortii** [comment]
> I am currently learning german so I can't filter out cards by proper nouns like it's possible with the japanese addon. A possible solution is for me to be able to add a list of proper nouns and Anki-Morphs to automatically skip them.

That is an amazing idea! I'll add it to the roadmap, and we will see if it is still necessary after I (hopefully) get spacy working. Thanks!

**Comment by mortii** [comment]
Added in #24
